README.md

```diff
@@ -121,6 +123,27 @@ rspec packs/foobar/spec
 rspec packs/foobar/nested_pack
 ```
 
+#### parallel_tests
+
+`parallel_tests` has it its own spec discovery mechanism, so Stimpack's RSpec integration doesn't do anything when you use them together.
+To make them work, you'll need to explicitly specify the spec paths:
+
+```bash
+RAILS_ENV=test bundle exec parallel_test spec packs/**/spec <other_options> rspec
```

Comment:
The last option is a value for `-t`:

```
    -t, --type [TYPE]                test(default) / rspec / cucumber / spinach
```

So it has to be

```bash
RAILS_ENV=test bundle exec parallel_test spec packs/**/spec <other_options> -t rspec
```

Or as the ordering does not matter, we could rearrange them to

```suggestion
RAILS_ENV=test bundle exec parallel_test spec packs/**/spec -t rspec <other_options>
```

Reply:
Thank you @oleg-vinted for pointing this out!
I haven't been using `parallel_test` so this was my (incorrect) guess 😅